crates/viewer/re_view/src/view_property_ui.rs

```diff
@@ -36,42 +36,29 @@ fn view_property_ui_impl(
     };
 
     let query_ctx = property.query_context(ctx);
-    // If the property archetype only has a single component, don't show an additional hierarchy level!
-    if reflection.fields.len() == 1 {
```

Comment:
The old code caused a UI showing "Time axis" followed by a checkbox, omiting the crucial "Link" title.

In general, I think we should avoid special cases like this in the UI. Better to be consistent, if sometimes extra verbose.

Reply:
Maybe we can omit it only if the name of the archetype and the component is identical ("Near clip plane > Near clip plane")?

> Maybe let's add the lock x axis option like we have on the y axis, that might be really nice and not hard to do!

I rather not, at least right now. The X axis is all the same for all the plots, but the Y axis may be very different, so on the Y axis it would make more sense to have multiple link groups.

Reply:
sounds good to me!

Reply:
I did something similar:

<img width="312" alt="Screenshot 2025-06-06 at 20 06 29" src="https://github.com/user-attachments/assets/8070fb8b-24e0-410b-a68b-18f3189da7c9" />